openshift_cli_installer/utils/gcp_utils.py

```diff
@@ -0,0 +1,71 @@
+import os
+import shutil
+from pathlib import Path
+import tempfile
+
+from simple_logger.logger import get_logger
+from openshift_cli_installer.utils.const import GCP_STR
+
+LOGGER = get_logger(name=__name__)
+
+
+def set_gcp_configuration(user_input):
+    """
+    Saves provided GCP Service Account file at location '~/.gcp/osServiceAccount.json'
+
+    If file already exists at this path, it will be copied to tmp file first.
+
+    Returns:
+        dict: A dictionary of parameters needed for setting GCP Service Account file.
+
+    """
+    if is_clusters_gcp_platform(user_input.clusters):
+        gcp_sa_file_dir = os.path.join(os.path.expanduser("~"), ".gcp")
+        openshift_installer_gcp_sa_file_path = os.path.join(gcp_sa_file_dir, "osServiceAccount.json")
+        gcp_params = {
+            "gcp_sa_file_dir": gcp_sa_file_dir,
+            "openshift_installer_gcp_sa_file_path": openshift_installer_gcp_sa_file_path,
+        }
+        gcp_params.update({"gcp_platform": True})
+
+        if os.path.exists(openshift_installer_gcp_sa_file_path):
+            backup_existing_gcp_sa_file_path = tempfile.NamedTemporaryFile(suffix="-installer.json").name
```

Comment:
set this direct to `backup_existing_gcp_sa_file_path` key

Reply:
`backup_existing_gcp_sa_file_path` is used below lines 33 and 35, that's why assigned it first and reused